config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -143,7 +143,7 @@ presubmits:
     # The script this job runs is not in all branches.
     - ^main$
     path_alias: sigs.k8s.io/cluster-api
-    run_if_changed: '^((api|bootstrap|cmd|config|controllers|controlplane|errors|exp|feature|hack|scripts|test|third_party|util)/|main\.go|go\.mod|go\.sum|Dockerfile|Makefile)'
+    run_if_changed: '^((api|bootstrap|cmd|config|controllers|controlplane|errors|exp|feature|hack|internal|scripts|test|third_party|util|webhooks)/|main\.go|go\.mod|go\.sum|Dockerfile|Makefile)'
```

Comment:
should we add also version?
also, wondering if we can less fragile by excluding the things that should be ignored (docs, logos, tilt_modules if I got it right)

Reply:
Yup, added version.

As far as I'm aware we can't invert the list. The problem is that it's just a regular Go regexp. I tried a few hours to get it to work in my old company and IIRC Go regexp doesn't support negative lookahead.

Reply:
thanks for checking